### ISSUE TITLE
catalog_get_department_details

### ISSUE THREAD

**Issue by elig0n** [zandoan/turing-backend#14]
doesn't return "department_id" like /departments/{id} example shows
is this a mistake ?

**Comment by edertaveiraturing** [comment]
No, because do you already have the ID when you pass it to the Stored Procedure.
